pkg/apis/provisioning/v1alpha5/provisioner.go

```diff
@@ -114,6 +114,10 @@ func (r Requirements) OperatingSystems() sets.String {
 	return r.Requirement(v1.LabelOSStable)
 }
 
+func (r Requirements) CapacityTypes() sets.String {
```

Comment:
To be consistent I'd add the same comment on the line above as per line 112 (eg)

Reply:
Would it offend you if I just remove all the other comments? I don't mean to be defensive, but I feel like I'm getting the theme of being too verbose in the feedback on this PR, so I feel like I'm gonna add that comment and then someone is going to tell me that it's a useless comment, and I'm going to agree with them =P

Reply:
The style is definitely to write self documenting code where possible and add comments where things are actually clear. I agree with you that these methods are basically just getters and could get by without comments.

Reply:
I'm gonna delete these comments then in my next commit then where I also (very sadly) revert all my doc updates =P